public/templates/demo.html

```diff
@@ -45,9 +43,35 @@
       href="/assets/css/owl-carousel.css"
     />
 
-    <!-- import the webpage's javascript file -->
+    <!-- Import the webpage's javascript file -->
+    <script defer src="/components/demo.js"></script>
+
+    <!-- jQuery -->
     <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-    <script src="/components/patientSignUp.js" defer></script>
+    <script
```

Comment:
Is jQuery needed in this page? If so, there seems to be two versions of jQuery being imported here IIUC:

- One from `code.jquery.com` that is 3.3.1
- One from `ajax.googleapis.com/.../jquery/` that is 1.7

Which one should be used?

Reply:
A lot of the bootstrap js is using Jquery to function. I deleted all the jquery imports and upgraded it to 3.5.1 version because it's the most recent.

Reply:
If we're going to change jQuery imports, we should probably do that in another issue and under another PR where we can do it for every page that needs the new import. It's better to do that kinda change in another PR so we can spot any functionality issue that might come up.

That said, I might be missing something but I don't see any `3.5.1` jQuery imports in this PR.

Reply:
Hey,

I hadn't had a chance to upload the latest pr!